### PR TITLE
allow require option to take a single file/function or Array

### PR DESCRIPTION
**Pull request by tteltrab**
modified the require option handling to allow for both a single file/function or an array of files/functions.  Gives more flexibility in what you can set before running tests.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ gulp.task('default', function () {
 
 ##### ui
 
-Type: `string`
```
Comment (shinnn) [inline readme.md]: All the double spaces in this API doc are explicitly added so you shouldn't remove them.

**Comment by sindresorhus** [comment]
No thanks

**Comment by tteltrab** [comment]
hey, any feedback as to what I'd need to do to get this as a standard feature? happy to write tests (though the other options aren't tested). Pulled from https://github.com/pghalliday/grunt-mocha-test as an example.  It is pretty powerful and doesn't regress the current functionality.

**Comment by mbarlock** [comment]
This is supported in Mocha, why would you not want it?

File: readme.md (hunk 1 above)
Comment (tteltrab) [inline readme.md]: apologies, editor did this automatically.  I'm curious, what's the purpose here?

Comment (shinnn) [inline readme.md]: For readability. Double spaces in Markdown will be converted to `<br>`.

[This rule is explicitly declared in our `.editorconfig`](https://github.com/sindresorhus/gulp-mocha/blob/80781186938661a6cc024536c2fa8ae05a9cb9ea/.editorconfig#L14-L15). You should use [EditorConfig](http://editorconfig.org/) plugin for your editor when you contribute to the project which includes an `.editorconfig` file.

Comment (tteltrab) [inline readme.md]: Thanks, thought I was but apparently not - easy to fix.  Doesn't seem like it much matters though, as this was outright closed with no justification.

**Comment by sindresorhus** [comment]
> hey, any feedback as to what I'd need to do to get this as a standard feature?

Don't really have anything else to say other than that I'm not interested in this. Too much.

> This is supported in Mocha, why would you not want it?

It's not afaik.

**Pull request closed without merging**